share/ezbackup_exclude.rsync

```diff
@@ -1,6 +1,8 @@
+- *.swp
+- *.swo
 - *.vdi
 - .cpanm/
 - tmp/
  *.gvfs
 - Perforce/
-- known_hosts
\ No newline at end of file
+- known_hosts
```

Comment:
I would prefer to leave known_hosts out of the defaults.

Reply:
I didn't add the known_hosts, I think my editor just added a trailing newline